aiohttp_sse/__init__.py

```diff
@@ -25,7 +29,7 @@ def hello(request):
 
     DEFAULT_PING_INTERVAL = 15
 
-    def __init__(self, *, status=200, reason=None, headers=None):
+    def __init__(self, *, status=200, reason=None, headers=None, request=None):
```

Comment:
I'm not convinced this is the right approach. Since we initialize the context manager outside the context manager interface. first we need to call `self.start(request)` before entering into the manager. It means we can't use `EventSourceResponse` as context manager without the `sse_response` wrapper. Since we are about to introduce a new API, let's just write a new sub class of `EventSourceResponse` that accept request as positional argument of __init__ ?
I will try that. let's see how it looks.